test/regression/sql/json_functions_duckdb.sql

```diff
@@ -0,0 +1,144 @@
+-- <JSON_EXISTS>
+-- Test 1: Path exists in a simple JSON object
+SELECT json_exists('{"a": {"b": 1}}', '$.a.b'); -- Expected: true
```

Comment:
I think most of these tests simply test that DuckDB works correctly, not so much the pg_duckdb intgration.

I think for each function it should give us enough confidence into pg_duckdb to add one test with reasonable arguments, and then tests with the various combinations of `NULL` arguments. So while I really appreciate the effort to add extensive tests, I don't think they add much value and would in practice only increase test runtime.

Reply:
Reduced the tests